<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>811</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Device</string>
     </attribute>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>179</x>
        <y>24</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_device">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add_camera">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_input_overlay">
      <property name="geometry">
       <rect>
        <x>357</x>
        <y>20</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Input</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overlay</string>
       </property>
      </item>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>741</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_inputs"/>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>741</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="layout_overlays"/>
     </widget>
     <widget class="QPushButton" name="pushButton_del_camera">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>338</x>
        <y>24</y>
        <width>31</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>as</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Template</string>
     </attribute>
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>721</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openfile">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>10</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Template</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_filename">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>16</y>
        <width>601</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>No template selected</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
